frontend/taipy-gui/src/components/Taipy/Input.tsx

```diff
@@ -103,20 +105,23 @@ const Input = (props: TaipyInputProps) => {
 
     return (
         <Tooltip title={hover || ""}>
-            <TextField
-                margin="dense"
-                hiddenLabel
-                value={value ?? ""}
-                className={className}
-                type={type}
-                id={id}
-                label={props.label}
-                onChange={handleInput}
-                disabled={!active}
-                onKeyDown={handleAction}
-                multiline={multiline}
-                minRows={linesShown}
-            />
+            <Box width={width} data-testid="textfield-box">
```

Comment:
You're changing the display mode of the input. 
Box is a div => display: block
We want the display to be more like a span (ie inline)

Reply:
I'm not sure anymore of what it would render 
@chialiotta Could you copy here a screenshot of a page with 2 input side by side ?
ie `<|{user_prompt_1}|input|width={500}|><|{user_prompt_2}|input|width=50%|>` ?

Reply:
![image](https://github.com/Avaiga/taipy/assets/55915226/8aaf7a2b-466d-4e8b-8fea-18550e0f18b5)

It is indeed not inline, I will change it. It might take me a few days to get to it though because I'm busy with my thesis, sorry